next/pages/form/[eform].tsx

```diff
@@ -60,6 +63,20 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   }
 }
 
+const initDefaultSchemaFields = (schema: StrictRJSFSchema) => {
```

Comment:
would rather for such functions to live within utils files if they are outside of the component - though I see this is presently unused so if it's temporal until we decide what to do with it ok to keep here (but mark it with TODO please in such case, so that it's easiliy visible for anyone doing a cleanup of TODOs on the project)